docs/agent-templates-ingestion.md

```diff
@@ -98,6 +98,74 @@ The system now supports multiple categories with specialized agent templates:
   - Focus: Financial planning, retirement strategies, tax optimization
   - Knowledge Base: `financial_planner_kb`
 
+#### 4. Legal Category
+
+The legal category provides comprehensive legal guidance across four specialized domains:
+
+**Family & Life Planning Suite** (`legal_family_life.yaml`):
+- **Divorce & Separation Specialist** (`divorce_separation_specialist`)
+  - Focus: Divorce procedures, property division, spousal support
+  - Model: `mistral:latest`
```

Comment:
This needs an update to be inline with the actual model. Also based on the other tempaltes not sure if mentioning the model is necessary here.